app/assets/javascripts/contact_me.js

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
I am not too familiar with bootstrap validations, but this is probably not the best place to put it. It's generic to the whole application. It should be exposed as a library or generic javascript file. Is there no gem that you can drop into your gemfile?  Maybe it's part of the bootstrap gem already and you just need to include it in your applications.js.